pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java

```diff
@@ -286,7 +286,7 @@ public static void buildSegmentFromAvro(File avroFile, TableConfig tableConfig,
       org.apache.pinot.spi.data.Schema schema, int segmentIndex, File segmentDir, File tarDir)
       throws Exception {
     // Test segment with space and special character in the file name
-    buildSegmentFromAvro(avroFile, tableConfig, schema, segmentIndex + " %", segmentDir, tarDir);
+    buildSegmentFromAvro(avroFile, tableConfig, schema, segmentIndex + "_", segmentDir, tarDir);
```

Comment:
Why do we need to change this? Is anything break because of this? We do want to test the case where segment name contains space and special character

Reply:
This is actually my bad. Was testing a few things. This is an issue when working with S3, but I'm fixing that separately here https://github.com/apache/pinot/pull/10055. Commits got mixed up. Reverted